### PR TITLE
Uncollect fix

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit dded8e03f7d52262b75561996892055cc656fb43:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/trackerbot.py [hunk 1]
```diff
@@ -273,9 +273,9 @@ def depaginate(api, result):
 def composite_uncollect(build):
     """Composite build function"""
 
-    url = "{}{}".format(conf['ostriz'], build)
+    url = "{}?build={}&source=jenkins".format(conf['ostriz'], build)
```
Comment (mfalesni) [inline utils/trackerbot.py]: shouldn't the `build` be escaped since it is a get parameter?

Comment (mfalesni) [inline utils/trackerbot.py]: https://docs.python.org/2/library/urllib.html#urllib.quote

**Comment by dajoRH** [comment]
Lint report for commit 796365c999acf49f35c8e45bb73f27c66d794b94:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**